Integrations/Cloaken/Cloaken.py

```diff
@@ -70,3 +73,39 @@ def get_client():
     else:
         # server error or unavailable
         return_error('Error Cloaken Unshorten: ' + str(response.get('data', 'key missing')))
+
+if demisto.command() == 'cloaken-screenshot-url':
+    client = get_client()
+    url = demisto.args()["url"]
+    screenshot = utility.RasterizeAndRetrieveImage(client, url=url)
+    context={"Url":url, "Status":'failed'}
+    try:
+        result = screenshot.get_screenshot()
+        stored_img = fileResult(result.get("filename","screenshot.png"), result.get("data","NA"), "image")
+        demisto.results({
```

Comment:
you should return the the file result to the war room.
```suggestion
        demisto.results(stored_img)
```

Reply:
ok

Reply:
Returning the file result did not work for me.  I had to revert this change.